cumulusci/tasks/salesforce/UpdateDependencies.py

```diff
@@ -305,7 +305,9 @@ def _install_dependency(self, dependency):
                     )
                 )
                 package_zip = InstallPackageZipBuilder(
-                    dependency["namespace"], dependency["version"]
+                    dependency["namespace"],
+                    dependency["version"],
+                    securityType=dependency.get("security_type", "FULL"),
```

Comment:
What do you think about making this a task option rather than a per-dependency option?

The reason I suggest that is that I'm imagining the automation in a project wanting to be able to run `update_dependencies` in different contexts with different behaviors, but without repeating the dependency list so that it can set this option. It seems more amenable to future refactorability.

I don't see much of a use case for projects wanting to specify a different security model for each package it installs on a _package_ basis, but rather on a task or flow basis. I _think_ it's more likely to be "install everything with X model" or, for regression, "install everything with X model and then run install_beta with Y model".

Reply:
@davidmreed good call.